src/plugins/plot/chart/MctChart.vue

```diff
@@ -52,6 +52,38 @@ const MARKER_SIZE = 6.0;
 const HIGHLIGHT_SIZE = MARKER_SIZE * 2.0;
 const ANNOTATION_SIZE = MARKER_SIZE * 3.0;
 const CLEARANCE = 15;
+const NO_HANDLING_NEEDED_ATTRIBUTES = {
+    label: 'label',
+    values: 'values',
+    format: 'format',
+    color: 'color',
+    name: 'name',
+    unit: 'unit'
+};
+const IMPLICIT_HANDLED_ATTRIBUTES = {
+    range: 'range',
+    //series stats update y axis stats
+    stats: 'stats',
+    frozen: 'frozen',
+    autoscale: 'autoscale',
+    autoscalePadding: 'autoscalePadding',
+    logMode: 'logMode',
+    yKey: 'yKey'
+};
+const HANDLED_ATTRIBUTES = {
```

Comment:
This is great! Could we just add a single sentence above each detailing the different categories?

Reply:
Done